newIDE/app/src/ProjectCreation/services/BrowserCreation.js

```diff
@@ -27,16 +29,22 @@ export const onCreateBlank = async ({
   sendNewGameCreated('');
 
   const project = gd.ProjectHelper.createNewGDJSProject();
-  return { project, storageProvider: null, fileMetadata: null };
+  return {
+    project,
+    storageProvider: null,
+    fileMetadata: { projectName, fileIdentifier: '' },
```

Comment:
@4ian I feel like it is a hack but making fileIdentifier optional calls for a big refactor. I don't know if the project name should be part of FileMetadata

Reply:
Nevermind, I just realized my code is nonsense